### ISSUE TITLE
Not downloading subtitles

### ISSUE THREAD

**Issue by Bert-Proesmans** [pannal/Sub-Zero.bundle#634]
Hi Pannal, first of all, thank you for the plugin. It always worked really well and I'm a fan of what you're doing!

I did a reinstall of Windows a while ago, reinstalled Plex and SubZero. I didn't port my old configuration but created new libraries pointing to my media files on a separate disk.
Since this event, I'm seeing new media either timing out (the UI spinner keeps spinning for at least 5 minutes) on receiving metadata or receiving them after a few refreshes. This also includes subtitles, which are never downloaded.
**edit:** My expectations are that it resolves all metadata and subtitles within a few (6-7) minutes after a library refresh. This matches my experience with previous installations.

I looked at the log files, attempted cache clears, purged Plex and reinstalled again but it doesn't seem to help. Permissions are ok. I waited for the issues to fix themselves for days in between and tried SZ-master-2019-04-14. Nothing changes and the logs tell me the same information every time.
The following is what I could understand from the SZ logs itself;
1. It times out on fetching resources;
2. It doesn't load subtitles when performing automated tasks, even though the plugin deck recognized that there are new media items eligible for checking.

Normally I (can) figure out this kind of stuff myself, but I'm stuck and don't know where to look anymore. I suppose the issue could be with the Plex install itself, I couldn't say for sure anymore. Can I ask you (the reader) to point me into a direction that will lead to a fix?

Attached are log files for the last startup of my server; ([Logs_190415_21-30-12.zip](https://github.com/pannal/Sub-Zero.bundle/files/3081838/Logs_190415_21-30-12.zip))
1. Set SZ logging level to Debug;
2. Plex server exit+start;
3. Refresh metadata on the latest added film on deck;
4. Waited 3 minutes;
5. Downloaded logs through plugin deck -> Subzero -> Advanced -> download logs.

I would really appreciate someone helping me out here. My eternal gratitude for the golden tip!

Thank you,
Bert

**Comment by pannal** [comment]
Hmm, I can't really see anything inside your log files that stands out. In your recent log I can't even see any calls being made to the plugin from the PMS. No metadata refresh calls or anything.

I'd start fresh again, if you've done all the setup steps from the wiki.

Also keep in mind: when you use the master checkout, it currently differs from the currently rolled-out one and gets automatically updated to the older one unless you set SZ to dev mode (see wiki).

The plugin repository has been removed from Plex, but the automatic updater still runs. I'm expecting the new stable release to hit this weekend.

**Comment by Bert-Proesmans** [comment]
I have to admit that I made a mistake in testing out the development version of SZ. (I guess I changed the dev-switch in the wrong file)

Just now I shut down Plex, downloaded SZ from the master branch, installed and changed the dev-switch. Restarted Plex and forced a library refresh. 
It all works again automatically; movies and tv shows with missing information instantly receive their proper data and subtitles in 10's of seconds. SZ also instantly wrote out two log rotations of the events ([Logs_190416_18-02-45.zip](https://github.com/pannal/Sub-Zero.bundle/files/3085844/Logs_190416_18-02-45.zip))
My issue appears to be solved now. I couldn't say for sure if it's because of SZ-DEV. There is no trace about resource hash retrieval timeouts anymore.

Thanks for taking the time to look at my case!
